libraries/corretto_helpers.rb

```diff
@@ -0,0 +1,39 @@
+module Java
+  module Cookbook
+    module CorrettoHelpers
+      def default_corretto_url(version)
+        if version.to_s == '8'
+          'https://corretto.aws/downloads/latest/amazon-corretto-8-x64-linux-jdk.tar.gz'
```

Comment:
if these are unversioned URLs won't this all break when a new release of 8 comes out and the sha doesn't match?

Reply:
yeah, and it's really annoying. 
Even worse, if the version changes, the folder name changes

Reply:
Do they not have an API we can call to get the sha at least?

Reply:
Doesn't look like they have an API to get the SHA's from, just tar containing all the SHAs, which seems even more wrong.